webpack.mix.js

```diff
@@ -6,6 +6,8 @@ let purgeCssOptions = {
   enabled: true,
   // List of regex of CSS class to not remove
   whitelistPatterns: [/^ball-pulse/],
+  whitelistPatterns: [/^vs__/],
```

Comment:
There is the reason ball-pulse was not working anymore:
```suggestion
  whitelistPatterns: [/^ball-pulse/, /^vs__/],
```